### PR TITLE
Feat 33 schema version

### PR DESCRIPTION
**Pull request by jtyoung84**
Closes #33

- Updates version of aind-data-schema to 0.33.3
- Separating the code out from this PR: https://github.com/AllenNeuralDynamics/aind-metadata-mapper/pull/38

### CODE REVIEW

File: src/aind_metadata_mapper/bergamo/session.py [hunk 1]
```diff
@@ -462,93 +463,99 @@ def _transform(self, extracted_source: RawImageInfo) -> Session:
...
+                                    ][1]["scanfields"]["duration"],
+                                    inter_spiral_interval=photostim_groups[0][
+                                        "rois"
+                                    ][2]["scanfields"]["duration"],
                                 ),
-                                number_spirals=int(
-                                    photostim_groups[0]["rois"][1][
-                                        "scanfields"
-                                    ]["repetitions"]
+                                PhotoStimulationGroup(
```
Comment (arielleleon) [inline src/aind_metadata_mapper/bergamo/session.py:512]: @jtyoung84 This looks like a duplicate

**Review by arielleleon — COMMENTED** [review]
Made one comment about a duplicate object

File: src/aind_metadata_mapper/bergamo/session.py (hunk 1 above)
Comment (jtyoung84) [reply src/aind_metadata_mapper/bergamo/session.py:512]: I think they're close, but there is a slight difference with the group index

**Review by arielleleon — APPROVED** [review]
Leaving as is until Kayvon and Marton finish the final draft with Saskia

**Pull request merged**